README.md

```diff
@@ -1 +1,3 @@
-# bulk-scan-orchestrator
\ No newline at end of file
+# bulk-scan-orchestrator
```

Comment:
Perhaps it is a good occasion to at least name it `Bulk Scan Orchestrator` 😉

Reply:
I will do the overall readme part in one of the coming PRs.